server/backends/chunkstore/chunkstore.go

```diff
@@ -23,11 +24,16 @@ type Chunkstore struct {
 	internalBlobstore    interfaces.Blobstore
 	writeBlockSize       int
 	writeTimeoutDuration time.Duration
+	noSplitWrite         bool
 }
 
 type ChunkstoreOptions struct {
 	WriteBlockSize       int
 	WriteTimeoutDuration time.Duration
+
+	// If true, individual buffers passed to Write will not be split accross
+	// multiple blobs unless the buffer itself exceeds the WriteBlockSize
+	NoSplitWrite bool
```

Comment:
This is still necessary; this prevents the chunkstore writer from splitting individual writes that are sent to it. Otherwise, the buffers from the terminal writer can be split across multiple chunks.